src/plugins/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.tsx

```diff
@@ -135,32 +152,46 @@ export function AiAssistantSelectionPage() {
                     <EuiSpacer size="s" />
                   </>
                 ) : null}
-                <EuiLink
-                  data-test-subj="securityAiAssistantSelectionPageDocumentationLink"
-                  external
-                  target="_blank"
-                  href="https://www.elastic.co/guide/en/security/current/security-assistant.html"
+                <p>
+                  {i18n.translate(
+                    'aiAssistantManagementSelection.aiAssistantSelectionPage.securityAssistant.documentationLinkDescription',
+                    { defaultMessage: 'For more info, see our' }
+                  )}{' '}
+                  <EuiLink
+                    data-test-subj="securityAiAssistantSelectionPageDocumentationLink"
+                    external
+                    target="_blank"
+                    href="https://www.elastic.co/guide/en/security/current/security-assistant.html"
+                  >
+                    {i18n.translate(
+                      'aiAssistantManagementSelection.aiAssistantSettingsPage.securityAssistant.documentationLinkLabel',
+                      { defaultMessage: 'Documentation' }
+                    )}
+                  </EuiLink>
+                </p>
+                <EuiButton
+                  data-test-subj="pluginsAiAssistantSelectionPageButton"
+                  iconType="gear"
+                  onClick={() =>
+                    navigateToApp('management', { path: 'kibana/securityAiAssistantManagement' })
+                  }
                 >
                   {i18n.translate(
-                    'aiAssistantManagementSelection.aiAssistantSettingsPage.securityAssistant.documentationLinkLabel',
-                    { defaultMessage: 'Documentation' }
+                    'aiAssistantManagementSelection.aiAssistantSelectionPage.securityAssistant.manageSettingsButtonLabel',
+                    { defaultMessage: 'Manage Settings' }
                   )}
-                </EuiLink>
+                </EuiButton>
               </div>
             }
             display="plain"
             hasBorder
-            icon={<EuiIcon size="l" type="logoSecurity" />}
+            icon={<EuiIcon size="xxl" type="logoSecurity" />}
             isDisabled={!securityAIAssistantEnabled}
-            layout="horizontal"
             title={i18n.translate(
               'aiAssistantManagementSelection.aiAssistantSelectionPage.securityLabel',
-              { defaultMessage: 'Elastic AI Assistant for Security' }
+              { defaultMessage: 'Elastic AI for Security' }
```

Comment:
@bojanasan was this intentional in the designs? I think we want `Assistant` here as this is how we refer to it in the [docs](https://www.elastic.co/guide/en/security/current/security-assistant.html), and is currently what O11y has for their card in settings as well.

Reply:
@spong, thank you yes. Lets leave the headings and labels as they are "Elastic AI Assistant for Security". I think we may consider changes but in the subsequent releases.